_includes/ee-linux-install-reuse.md

```diff
@@ -137,56 +137,60 @@ You only need to set up the repository once, after which you can install Docker
 
 {% elsif section == "install-using-yum-repo" %}
 
-There are currently two versions of Docker EE Engine available:
+The current version of Docker EE Engine is 18.09 for Docker Enterprise Edition 2.1.
```

Comment:
The current version of Docker EE Engine is 18.09 for use with Docker Enterprise Edition 2.1